.env.template

```diff
@@ -0,0 +1,5 @@
+SSO_JWT_SECRET=jwt-secret
+SSO_COOKIE_NAME=tdc_auth_token
+DEV_HOST=campaign-api.lvh.me
+DISCOURSE_LOGIN_URL=http://lvh.me:3000/session/sso_cookies
```

Comment:
Why we need this two URLs here @orlando ?

Reply:
For no special reason, just to keep them separated and easy to change.